crates/ruff_python_ast/src/visitor/preorder.rs

```diff
@@ -805,6 +833,24 @@ where
     }
 }
 
+pub fn walk_type_param<'a, V>(visitor: &mut V, type_param: &'a TypeParam) where
+V: PreorderVisitor<'a> + ?Sized,
+{
+    match type_param {
+        TypeParam::TypeVar(TypeParamTypeVar {
+            bound,
+            name: _,
+            range: _,
+        }) => {
+            if let Some(expr) = bound {
+                visitor.visit_expr(expr);
+            }
+        }
+        TypeParam::TypeVarTuple(_) => {}
+        TypeParam::ParamSpec(_) => {}
```

Comment:
These are just contain an identifier and range so there is nothing to visit.

Reply:
Sounds right to me.